File: soft2/lec01/test0.c
/* test0.c */
#include <stdio.h>

int test(int i, int j){
  return (i * j);
}

int main(int argc, char *argv){
  int i,j,k;
  i = 3;
  j = 2;
  k = test(i, j);
  if (k > 5) printf(">5\n");
  else printf("<=5\n");
  return 0;
}

File: soft2/lec01/test0.java
class test0 {
    public static int test(int i, int j){
	return (i * j);
    }
    public static void main(String args[]){
	int i,j,k;
	i = 3;
	j = 2;
	k = test(i,j);
	if(k>5){
	    System.out.println(">5");
	} else {
	    System.out.println("<=5");
	}
    }
}
